src/relay/transforms/fake_quantization_to_integer.cc

```diff
@@ -148,7 +93,8 @@ class SubgraphExtractor : public ExprVisitor {
   const AffineTypeMap GetAffineTypes() { return affine_types_; }
   void VisitExpr(const Expr& expr) override {
     if (expr.as<CallNode>() == nullptr && expr.as<OpNode>() == nullptr &&
```

Comment:
can you add a comment explaining this check and why it implies `fake_quantization = false`? It's a little confusing to me.